src/System.IO.Pipelines/src/System/IO/Pipelines/FlushResult.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.IO.Pipelines
+{
+    /// <summary>
+    /// Result returned by <see cref="PipeWriter.FlushAsync"/> call
+    /// </summary>
+    public struct FlushResult
+    {
+        internal ResultFlags ResultFlags;
+
+        /// <summary>
+        /// Creates a new instance of <see cref="FlushResult"/> setting <see cref="IsCanceled"/> and <see cref="IsCompleted"/> flags
```

Comment:
Use paramref instead of cref when referring to method arguments. Here and elsewhere.

Reply:
There are no references to method arguments on this line.